src/app/shared/modal/modal.component.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Component,
+  OnInit,
+  ElementRef,
+  Output,
+  EventEmitter,
+} from '@angular/core';
+
+@Component({
+  selector: 'app-modal',
+  templateUrl: './modal.component.html',
+  styleUrls: ['./modal.component.scss'],
+})
+export class ModalComponent implements OnInit {
+  @Output() close = new EventEmitter();
+  @Output() clearConfirm = new EventEmitter();
+  constructor(private ele: ElementRef) {
+    console.log(ele.nativeElement);
+  }
+
+  ngOnInit(): void {
```

Comment:
Also you have to implement `OnDestroy` to remove the modal angular component element from `body` element

Reply:
implement it but I think no effect when implement it .